koku/masu/management/commands/migrate_trino_tables.py

```diff
@@ -420,46 +445,50 @@ def find_expired_partitions(schema, months, table, source_column_param):
             {source_column_param} as source
         FROM  "{table}$partitions"
     ) as partitions
-    WHERE partitions.partition_date < DATE '{expiration_date_param}'
+    WHERE partitions.partition_date < DATE '{expiration_date.date()!s}'
     GROUP BY partitions.year, partitions.month, partitions.source
         """
-    LOG.info(f"Finding expired partitions for {schema} {table}")
-    return run_trino_sql(expired_partitions_query, schema)
+    LOG.info(f"{prefix}Finding expired partitions for {schema} {table}")
```

Comment:
super nitpick: 

since `prefix` already contains the `schema`, is it necessary here too?

Reply:
Nope, it's redundant.